blockbin/src/App.css

```diff
@@ -0,0 +1,33 @@
+.App {
```

Comment:
convention for CSS classes is generally lower case (`.app`)

Reply:
hmm boilerplate was otherwise but i trust u